locales/en/fields.json

```diff
@@ -215,6 +215,19 @@
   "special_vehicle": {
     "label": "Special vehicle details"
   },
+  "special_vehicle_check": {
+    "label": "$t(fields::special_vehicle__explicit.label)",
+    "label_with_error": "$t(fields::special_vehicle__explicit.label_with_error)",
+    "hint": "$t(fields::special_vehicle__explicit.hint)",
+    "items": {
+      "yes": {
+        "label": "Yes, they must be held in custody"
```

Comment:
Is this content correct? Feels like this is more linked to release status.

Reply:
Er, it's straight up wrong. I copied and pasted the wrong thing